<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="hsrb"/>
  <arg name="debug" default="false"/>
  <arg name="gui" default="true" />
  <arg name="rviz" default="true"/>
  <arg name="gazebo_visualization" default="false" />
  <arg name="use_manipulation" default="true" />
  <arg name="use_navigation" default="true" />
  <arg name="use_perception" default="true" />
  <arg name="use_task" default="true" />
  <arg name="use_teleop" default="true" />
  <arg name="use_web" default="true" />
  <arg name="use_laser_odom" default="true" />
  <arg name="paused" default="true" />
  <arg name="fast_physics" default="false" />
  <arg name="highrtf" default="false" />
  <arg name="with_handle" default="true" />
  <arg name="use_oss_stacks" default="false" />
  <arg name="seed" default="1" />

  <include file="$(find hsrb_wrs_gazebo_launch)/launch/include/wrs_common.xml" pass_all_args="true" >
    <arg name="per_category" value="0" />
    <arg name="obstacles" value="0" />
    <arg name="per_row" value="0" />
  </include>

  <node name='spawn_orange' pkg='gazebo_ros' type='spawn_model' args='-file $(find tmc_wrs_gazebo_worlds)/models/ycb_017_orange/model-1_4.sdf -sdf -model orange -x -0.3 -y 0.2 -z 0.5' />

  <node name='spawn_soccer_ball' pkg='gazebo_ros' type='spawn_model' args='-file $(find tmc_wrs_gazebo_worlds)/models/ycb_053_mini_soccer_ball/model-1_4.sdf -sdf -model soccer -x 2.70 -y -0.8 -z 0.5' />

  <node name='spawn_baseball' pkg='gazebo_ros' type='spawn_model' args='-file $(find tmc_wrs_gazebo_worlds)/models/ycb_055_baseball/model-1_4.sdf -sdf -model baseball -x 2.70 -y -0.95 -z 0.5' />

  <node name='spawn_softball' pkg='gazebo_ros' type='spawn_model' args='-file $(find tmc_wrs_gazebo_worlds)/models/ycb_054_softball/model-1_4.sdf -sdf -model softball -x 2.70 -y -1.15 -z 0.5' />

</launch>
